### ISSUE TITLE
Idle Disconnect after 1 Minute

### ISSUE THREAD

**Issue by magic-gd** [drtwisted/godot-twicil#9]
Hi,
I noticed if there is no activity in the twitch chat, my TwiCIL Node gets disconnected after exactly 1 minute.
This can easily be reproduced and seen using the log timestamps.
Is there any way to prevent this?

![disconnect](https://user-images.githubusercontent.com/63213481/97086206-2b888b00-1622-11eb-959b-e42e7d54e75a.png)

**Comment by x11joe** [comment]
I agree, this is happening to me also! Very annoying.  I'm studying the code now to see if there is anything I can suggest as a modification.

**Comment by x11joe** [comment]
I should note that, I don't see the disconnected message like you.  It just simply stops working.  I'm testing now to see when it just decides to stop working.  It seems almost random for me at the moment, but always after 25 minutes.

**Comment by x11joe** [comment]
After testing today, it does not disconnect at all now??, I'm happy but so confused since I don't know what caused it in the first place yet.  Very strange, I even ran it over night and it never disconnected.  My friend tested it and it disconnected after 4 hours.  So it seems to be random.  Makes me wonder if there is a way to detect the disconnection and attempt an auto reconnect?
